tests/unit/test_data_storage.py

```diff
@@ -31,6 +31,7 @@
 
 @pytest.mark.parametrize("tree", [COMPLEX_TREE], indirect=True)
 def test_dir_expansion(cloud_test_catalog, version_aware, cloud_type):
+    pytest.skip("Skipping as dir expansion must be re-implemented in application level")
```

Comment:
This test is quite useful. Without it, I fear that the dir expansion code willl bitrot very quickly.

Reply:
Ok, returned it. Needed to add simple util function to "old" schema with rooted file signals as current implementation of dir expansion only works with those and legacy CLI code that's still using them.